src/uu/ls/src/ls.rs

```diff
@@ -63,13 +70,13 @@ Ignore files and directories starting with a '.' by default"#;
 const USAGE: &str = "{} [OPTION]... [FILE]...";
 
 pub mod options {
-
     pub mod format {
         pub static ONE_LINE: &str = "1";
         pub static LONG: &str = "long";
         pub static COLUMNS: &str = "C";
         pub static ACROSS: &str = "x";
-        pub static TAB_SIZE: &str = "tabsize"; // silently ignored (see #3624)
+        pub static TAB_SIZE: &str = "tabsize";
+        // silently ignored (see #3624)
```

Comment:
please revert this or move the comment above.
it gives the impression that it applies to COMMAS